ee/src/main/java/io/supertokens/ee/EEFeatureFlag.java

```diff
@@ -94,7 +93,7 @@ public void updateEnabledFeaturesValueReadFromDbTime(long newTime) {
     public void constructor(Main main, AppIdentifier appIdentifier) {
         this.main = main;
         this.appIdentifier = appIdentifier;
-        Cronjobs.addCronjob(main, EELicenseCheck.getNewInstance(main, this.appIdentifier.getAsPublicTenantIdentifier()));
+        EELicenseCheck.getOrCreateInstance(main);
```

Comment:
remove this from here and put it in the Main.java like the other cronjobs

Reply:
This is task class is not accessible from main